python/semantic_kernel/orchestration/sk_function_base.py

```diff
@@ -119,11 +143,32 @@ async def invoke_async(
         pass
 
     @abstractmethod
-    async def invoke_with_custom_input_async(
+    def invoke_with_vars(
         self,
         input: ContextVariables,
-        memory: SemanticTextMemoryBase,
-        skills: "ReadOnlySkillCollectionBase",
+        memory: Optional[SemanticTextMemoryBase] = None,
```

Comment:
Either missing a skills input argument or skills is not needed

Reply:
the "skills" param was a bug, that's why I removed it

Reply:
fixed